homeassistant/components/traccar/strings.json

```diff
@@ -7,7 +7,7 @@
       }
     },
     "abort": {
-      "one_instance_allowed": "Only a single instance is necessary.",
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
```

Comment:
For webhook flows the key needs to be `one_instance_allowed`

Reply:
Oh okay did not know that, will close this PR and open another one to add this string to the base strings.

Reply:
Probably better to change the Base class for webhook flows to use the same as others ;)

Consistency :tada:

Reply:
That is an even better idea :smile: 
I am a big fan of consistency as well :confetti_ball: